test/pub_verif_token.test.ts

```diff
@@ -51,6 +52,18 @@ async function keysFromVector(v: Vectors): Promise<[CryptoKeyPair, Uint8Array]>
     return [{ privateKey, publicKey }, publicKeyEnc];
 }
 
+class TestSerializeTokenRequest {
```

Comment:
no need to use this class if you use this call, 
```
testSerializeType(TOKEN_TYPES.BLIND_RSA, TokenRequest, tokReq);
```
and change in ./src/pub_verif_token.ts
```diff
-        tokenType: BlindRSAType | PartiallyBlindRSAType,
+        tokenType: TokenTypeEntry,
```
and validate the type in the constructor.

Reply:
we need a new class for typing reason. `deserialize` of TokenRequest requires a `TokenTypeEntry` parameter, and therefore the signature does not match the expected `CanDeserialize`
see the typescript error
```
Type '(bytes: Uint8Array, tokenType: TokenTypeEntry) => TokenRequest' is not assignable to type '(type: TokenTypeEntry, _b: Uint8Array) => CanSerialize'.
```

Reply:
EDIT: I've changed the function signature to be `(TokenTypeEntry, Uint8Array) => TokenRequest`, and this matches `testSerializeType` definition. Ordering needed to change.